internal/test/credentials/credentials.go

```diff
@@ -0,0 +1,80 @@
+package credentials
+
+import (
+	"fmt"
+	"golang.org/x/crypto/ssh"
+	"strings"
```

Comment:
Let us go with @LorbusChris suggestion to group imports with new files. Note: I am not asking you to make this change in existing files.